### DISCUSSION TITLE
v4l backend live image reverse

### DISCUSSION THREAD

**Discussion by colaplate** [photobooth-app/photobooth-app#69] [Howtos and camera specific topics]
My webcam has been configured and working normally in v4l backend, and I want to know how to make the live image reverse left and right

**Comment by mgineer85** [comment]
Hi, you mean flip horizontally? Currently this is implemented for opencv backend, but not yet for v4l2py backend. Can you switch so opencv? I can check later over the weekend whether v4l2py has this option also

**Reply by colaplate** [reply]
To be honest, my English is not very good, I am very sorry, thank you for your help, because I am using a webcam as the live view  at the backend, but the screen is reversed, I think it should be like looking in a mirror. have a nice day and thank you
